style.css

```diff
@@ -1006,3 +1093,1378 @@
+    display: flex;
+    flex-direction: column;
+    position: relative;
+  }
+
+  .first-page-intro {
+    margin-left: 89.01px;
+  }
+
+  .section h1 {
+    margin-right: 731px;
+    width: 327px;
+    height: 124px;
+    font-family: var(--font3);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 56px;
+    line-height: 64px;
+    letter-spacing: 0.37px;
+    color: #ff6b00;
+  }
+
+  .section .title {
+    margin-right: 731px;
+    width: 327px;
+    height: 27px;
+    font-family: var(--font2);
+    font-style: normal;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+    display: flex;
+    align-items: center;
+    color: #fff5e1;
+  }
+
+  .section .para {
+    width: 327px;
+    height: 123px;
+    font-family: var(--font1);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #fff;
+    margin-bottom: 12px;
+  }
+
+  .section ul {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    width: 20px;
+    height: 120px;
+    top: 35%;
+    justify-content: space-between;
+    position: absolute;
+  }
+
+  .section ul li {
+    list-style: none;
+  }
+
+  .page2 {
+    height: 1805px;
+    width: 1204px;
+    position: relative;
+    margin: auto;
+  }
+
+  .desktop-page2 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 1133px;
+    height: 52px;
+    margin-left: 35.5px;
+    margin-right: 35.5px;
+    position: absolute;
+    top: 10%;
+  }
+
+  .main {
+    width: 1156px;
+    height: 1379px;
+    margin-top: 301px;
+  }
+
+  .div-page2 {
+    display: flex;
+    flex-direction: row;
+    width: 1156px;
+    height: 448px;
+    padding: 0;
+    gap: 24px;
+  }
+
+  .myworks {
+    width: 327px;
+    height: 83px;
+    font-family: var(--font3);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #1c1a19;
+  }
+
+  .line {
+    width: 327px;
+    height: 0;
+    border: 1px solid #212121;
+    margin-top: 24px;
+  }
+
+  .desktop-page2 .line {
+    width: 794.3px;
+    margin: auto;
+  }
+
+  .image-desktop {
+    height: 448px;
+    width: 684px;
+    background-image: url(images/Img_Placeholder_desktop_first.png);
+    background-repeat: no-repeat;
+  }
+
+  .div-page2 .image {
+    display: none;
+  }
+
+  .details {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    width: 448px;
+    height: 348px;
+  }
+
+  .story {
+    width: 327px;
+    height: 36px;
+    font-family: var(--font3);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #091e42;
+    margin-top: 24px;
+  }
+
+  .description {
+    width: 327px;
+    height: 147px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    margin-top: 24px;
+  }
```

Comment:
- [ ] Some of your project cards are not being displayed also. PLease have a look at the screenshot taken below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/188199358-15e441e5-ebf4-4af0-a280-bef3497bb7f3.png)

I really don't what happened. I am running your project locally but I don't know what happened with your images and styles so please check them again and fix them 🏃